scripts.js

```diff
@@ -1,84 +1,110 @@
+const storeApp = {};
 
-// this is the store data. normally we would be grabbing data from elsewhere (an external api), but this will represent the data model! 
-  const totalInventory = [
-    {
-      title: 'Bowie Tee',
-      url: 'images/bowie.jpg',
-      price: 19.99,
-      stock: 4,
-    }, 
-    {
-      title: 'Don\'t Know Tee',
-      url: 'images/dontevenknow.jpg',
-      price: 22.50,
-      stock: 8,
-    }, 
-    {
-      title: 'Doughnut Jean Jacket',
-      url: 'images/doughnut.jpg',
-      price: 59.00,
-      stock: 5,
-    }, 
-    {
-      title: 'Journey Tee',
-      url: 'images/journey.jpg',
-      price: 22.99,
-      stock: 6,
-    }, 
-    {
-      title: 'Skeleton Jean Jacket',
-      url: 'images/someurl.jpg',
-      price: 30.00,
-      stock: 0,
-    }, 
-    {
-      title: 'Skeleton Hand Tee',
-      url: 'images/skeleton.jpg',
-      price: 30.00,
-      stock: 10,
-    }, 
-    {
-      title: 'HackerYou Hoodie',
-      price: 50.00,
-      stock: 4,
-    }, 
-  ]
+storeApp.totalInventory = [
+  {
+    title: 'Bowie Tee',
+    url: 'images/bowie.jpg',
+    price: 19.99,
+    stock: 4,
+  },
+  {
+    title: 'Don\'t Know Tee',
+    url: 'images/dontevenknow.jpg',
+    price: 22.50,
+    stock: 8,
+  },
+  {
+    title: 'Doughnut Jean Jacket',
+    url: 'images/doughnut.jpg',
+    price: 59.00,
+    stock: 5,
+  },
+  {
+    title: 'Journey Tee',
+    url: 'images/journey.jpg',
+    price: 22.99,
+    stock: 6,
+  },
+  {
+    title: 'Skeleton Jean Jacket',
+    url: 'images/someurl.jpg',
+    price: 30.00,
+    stock: 0,
+  },
+  {
+    title: 'Skeleton Hand Tee',
+    url: 'images/skeleton.jpg',
+    price: 30.00,
+    stock: 10,
+  },
+  {
+    title: 'HackerYou Hoodie',
+    price: 50.00,
+    stock: 4,
+  },
+]
 
-  // document ready function allows page to load before running any of our scripts 
-$(function(){
+storeApp.currencies = {
+  usd: {
+    exchange: 1,
+    symbol: `$`,
+    displayName: `USD`,
+    flag: `images/USD-flag.png`
+  },
+  cad: {
+    exchange: 1.28,
+    symbol: `$`,
+    displayName: `CAD`,
+    flag: `images/CAD-flag.png`
+  },
+  gbp: {
+    exchange: 0.76,
+    symbol: `£`,
+    displayName: `GBP`,
+    flag: `images/GBP-flag.png`
+  }
+}
 
-  // this object allows us to organize some information that we want to display conditionally depending on what currency the user selects
+storeApp.currentStock = storeApp.totalInventory.filter((item) => {
```

Comment:
I noticed there's an extra bracket around 'item' that wraps to the end of the arrow function - is that intended? I'm not sure that would work as-expected because (I believe) this means your function is looking at the whole function body like it's a parameter.

Reply:
storeApp.currentStock = storeApp.totalInventory.filter(item) => { return item.url && item.stock;
```suggestion
storeApp.currentStock = storeApp.totalInventory.filter((item) => {
```
};